lib/index.js

```diff
@@ -69,25 +74,9 @@ function configure() {
   if('cert' in bedrock.config.server) {
     httpsOptions.cert = fs.readFileSync(bedrock.config.server.cert);
   }
-  let caFiles = bedrock.config.server.ca;
-  if(typeof caFiles === 'string' ||
-    (Array.isArray(caFiles) && caFiles.length > 0)) {
-    if(!Array.isArray(caFiles)) {
-      caFiles = [caFiles];
-    }
-    // all certs must be parsed individually
-    httpsOptions.ca = [];
-    caFiles.forEach(file => {
-      let bundle = fs.readFileSync(file);
-      try {
-        const certs = forge.pem.decode(bundle);
-        bundle = certs.map(cert => forge.pem.encode(cert));
-      } catch(e) {
-        throw new Error(e.message);
-      }
-      httpsOptions.ca.push.apply(httpsOptions.ca, bundle);
-    });
-  }
+  const caFiles = bedrock.config.server.ca;
+  _parseCerts({caFiles, httpsOptions});
```

Comment:
This helper should return the parsed CAs as an array rather than passing in `httpsOptions` and updating it internally. Returning the parsed CAs is cleaner and more obvious code than modifying the internals of a variable someone passes into a function.